### DISCUSSION TITLE
Breaking change in Django 4.2

### DISCUSSION THREAD

**Discussion by Scotchester** [encode/django-rest-framework#8539] [Potential Issue]
Hi folks,

The [Wagtail](https://github.com/wagtail/wagtail) project has a check against Django's `main` branch in our CI, and it started failing a day or two ago due to [the removal of `django.http.multipartparser.parse_header()`](https://github.com/django/django/pull/15797), which [DRF uses in three places](https://github.com/encode/django-rest-framework/search?q=parse_header). This change will be [released in Django 4.2](https://github.com/django/django/pull/15804#issuecomment-1169125386).

[The recommended solution](https://docs.djangoproject.com/en/dev/releases/4.2/#miscellaneous) is to switch to `django.utils.http.parse_header_parameters()`. Shall I open an issue for this?

**Comment by lovelydinosaur** [comment]
Yup that sounds sensible. Thanks!

**Reply by carltongibson** [reply]
Resolved by #8556.

**Reply by Scotchester** [reply]
Thanks, folks!
